bigframes/display/anywidget.py

```diff
@@ -160,15 +167,17 @@ def _get_next_batch(self) -> bool:
             batch = next(iterator)
             self._cached_batches.append(batch)
             return True
-        except StopIteration:
+        except (StopIteration, ValueError) as e:
             self._all_data_loaded = True
+            if isinstance(e, ValueError):
+                # If we fail to get a batch, assume no more data is available.
+                self.row_count = 0
```

Comment:
We should not hide such errors. `ValueError` indicates some bug in BigFrames or the BigQuery API. The user needs to know about those.